src/digma/templates/debug-services.yaml

```diff
@@ -16,6 +16,15 @@ spec:
         app: "digma-debug-services"
     spec:
       containers:
+
+      - name: kafka-ui
+        image: provectuslabs/kafka-ui:v0.7.1
+        env:
+        - name: KAFKA_CLUSTERS_0_NAME
+          value: NTNlOGNmNjlkZjY4NDkxOT
```

Comment:
I think we should have different value, so it would be different from the value on our local dev machines